rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -214,8 +214,8 @@ maas_fqdn_extension:
 # maas_filesystem_threshold: Sets the threshold (%) for filesystem monitoring when you are not
 #                            specifying specific filesystems.
 #
-maas_filesystem_warning_threshold: 80.0
-maas_filesystem_critical_threshold: 90.0
+maas_filesystem_warning_threshold: 70.0
+maas_filesystem_critical_threshold: 80.0
```

Comment:
Is it possible to make this change only for ceph osd? Not sure if it's a good idea across the board.